src/XMakeCommandLine/XMake.cs

```diff
@@ -1316,19 +1316,21 @@ internal static void SetConsoleUI()
 #if FEATURE_GET_COMMANDLINE
             // split the command line on (unquoted) whitespace
             ArrayList commandLineArgs = QuotingUtilities.SplitUnquoted(commandLine);
-            // discard the first piece, because that's the path to the executable -- the rest are args
-            s_exeName = FileUtilities.FixFilePath(QuotingUtilities.Unquote((string)commandLineArgs[0]));
+            
+            s_exeName = FileUtilities.FixFilePath(QuotingUtilities.Unquote((string)commandLineArgs[0
```

Comment:
This can't be right. Make sure you switch configurations back to Debug and try this, it definitely shouldn't compile. Or am I missing something in the #if's?